pipeline/data_merging/bayesdel/convert_merged_variants_to_vcf.py

```diff
@@ -0,0 +1,45 @@
+import click
+import pandas as pd
+
+
+def _coord_to_vcf_cols(coord):
+    return (coord.split(':')[0].lstrip('chr'),  # chromosome col
+            int(coord.split(':')[1].lstrip('g.')),  # pos col
+            coord,  # id col
+            coord.split(':')[2].split('>')[0],  # ref col
+            coord.split(':')[2].split('>')[1])  # alt col
+
+
+def convert_merged_to_vcf(path, vcf_path):
+    coord_col = 'Genomic_Coordinate_hg38'
+
+    df = pd.read_csv(path, sep='\t')
+
+    vcf_cols = df.apply(lambda d: _coord_to_vcf_cols(d[coord_col]), axis=1)
+
+    df_vcf = pd.DataFrame.from_dict(dict(zip(vcf_cols.index, vcf_cols.values))).T
+    df_vcf.columns = ['CHROM', 'POS', 'ID', 'REF', 'ALT']
+    df_vcf['QUAL'] = df_vcf['FILTER'] = df_vcf['ID'] = '.'
+    df_vcf['INFO'] = '.'
+
+    # filtering out variants with unambiguous bases
```

Comment:
clarification -- isn't this filtering out variants with ambiguous bases? not unambiguous bases.